nextflow/nextflow_examples.ipynb

```diff
@@ -0,0 +1,1108 @@
+    "The nf_core RNASeq pipeline produces a large volume of informative output.\n",
+    "When inspecting the output, take note each time the executor you configured, the Google Life Sciences API, is invoked to execute the workflow steps that comprise RNASeq, from initial processing to alignment and creation of BAM files to creation of the MultiQC output files.\n",
+    "\n",
+    "When the pipeline run has completed successfully, the last few lines of output should resemble:\n",
+    "```\n",
+    "Waiting for file transfers to complete (1 files)\n",
+    "-[nf-core/rnaseq] Pipeline completed successfully-\n",
+    "Completed at: DD-Month-YYY HH:SS:MM\n",
+    "Duration    : 58m 56s\n",
+    "CPU hours   : <HOURS>\n",
+    "Succeeded   : <NUBMER of TASKS>\n",
+    "\n",
+    "Restoring the original gcloud project configuration: <GOOGLE-CLOUD-PROJECT-ID>\n",
+    "Updated property [core/project].\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e266fbc3-f018-42a4-9b18-9c3dec3a1f56",
+   "metadata": {
+    "tags": [
+     "\"output_scroll\"",
+     "\"scroll\":\"true\""
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "!terra nextflow -c /home/jupyter/nextflow.config run nf-core/rnaseq -profile google -w $(terra resource resolve --name=ws_files_autodelete_after_two_weeks)/nf_core/wd -r 3.11.1 \\\n",
+    "--minAssignedFrags 1 \\\n",
+    "--outdir $(terra resource resolve --name=ws_files_autodelete_after_two_weeks)/nf_core/output \\\n",
+    "--input /home/jupyter/test-datasets/samplesheet/samplesheet_minimal.csv \\\n",
+    "--genome 'R64-1-1'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e8257646-3704-4d75-9af0-141bf5266a89",
+   "metadata": {},
+   "source": [
+    "### Optional: Resume pipeline execution from cached data\n",
+    "\n",
+    "Nextflow pipelines can be run with the [`--resume` flag](https://www.nextflow.io/docs/latest/getstarted.html#modify-and-resume). This flag causes the pipeline run to rely on existing, cached intermediate results for some tasks; computation of new intermediate results is performed only where inputs or the pipeline have been modified. This feature allows greater efficiency and speed in pipeline development and debugging.\n",
+    "\n",
+    "To resume pipeline execution for a specific run, provide the session ID of the desired run following the `--resume` flag in the cell below. If no modifications have been made to the pipeline or its configuration, resuming a successful run should take about 20 minutes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a48fdef8-2140-40fe-9686-37fd67d02285",
+   "metadata": {
+    "tags": [
+     "\"output_scroll\""
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "!terra nextflow -c /home/jupyter/nextflow.config run nf-core/rnaseq -resume <SESSION_ID> -profile google -w $(terra resource resolve --name=ws_files_autodelete_after_two_weeks)/nf_core/wd -r 3.11.1 \\\n",
```

Comment:
With the 'resume' (which I've never actually tested until now :), I see the following error.  Does this mean that the outdir should be changed to a new path for the 'resume' run?
![CleanShot 2023-04-25 at 11 19 55@2x](https://user-images.githubusercontent.com/115093/234367294-9d264296-3e0b-4c0e-b26d-cecab21b3b5a.png)

Reply:
I've added content explaining this behavior, which is a known Nextflow bug, and validated that changing the output directory resolves this issue, as well as providing a different output directory in the command. See [screenshot](https://screenshot.googleplex.com/8pUW9omemGRFgzh) for validation that using a different directory results in successful execution/no warning.

Reply:
If you just change the outdir in the 'resume' example (e.g. change it to `.../nf_core/output_resume` or something, what happens?